strongbox-data-service/pom.xml

```diff
@@ -135,6 +135,17 @@
             <artifactId>commons-beanutils</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-client</artifactId>
+            <version>${version.hazelcast}</version>
+        </dependency>
+        <dependency>
+            <groupId>com.hazelcast</groupId>
+            <artifactId>hazelcast-spring</artifactId>
+            <version>${version.hazelcast}</version>
```

Comment:
Cosmetic: Could you please drop the version here, as it's inherited from the `strongbox-parent`?